### ISSUE TITLE
Does Projectknife enable email notifications for task assignments to users?

### ISSUE THREAD

**Issue by jvigil48** [projectknife/pkg-projectknife#44]
I have been working to get PK up and running on my site and noticed that no email notifications are going out to users on my test task assignments.  Is this function enabled in the version like it was in in Projectfork?

**Comment by angek** [comment]
Yes it does. Notifications are provided via a content plugin. Make sure the plugin in enabled and configured correctly.

**Comment by jvigil48** [comment]
Thanks Angek.  I am looking for the plugin but don't see it.  In PF the plugin was named "Projectfork Notifications" but I don't see a plugin like that for PK.  Is the plugin part of the base install or with a different mod or component?

**Comment by angek** [comment]
@jvigil48, you are right. It doesn't exist for Projectknife. Looks like my Projectfork installation is taking care of them. Will look into how quickly I can write it up for Projectknife. Should have time early nearly week.

**Comment by jvigil48** [comment]
Thank you Angek.  I am going to send you an email to your personal account regarding this.

**Comment by angek** [comment]
got it... have replied
